ambry-cloud/src/main/java/com/github/ambry/cloud/CloudRequestAgent.java

```diff
@@ -86,7 +86,8 @@ private void throwOrDelay(Throwable e, String actionName, String partitionPath,
         vcrMetrics.retryWaitTimeMsec.inc(delay);
       } else {
         // Either not retryable or exhausted attempts.
-        logger.error("{} failed partition {} statusCode {} cause {} made {} attempts.", actionName, partitionPath,
+        // Not very useful to log error here as they are printed in several other places
```

Comment:
It'll hide all exceptions.
If you want to avoid ttl updated exception log, please do it for the ttl update exception. Not all exceptions.

Reply:
done